src/passes/Print.cpp

```diff
@@ -1955,6 +1959,8 @@ struct PrintSExpression : public OverriddenVisitor<PrintSExpression> {
   Function::DebugLocation lastPrintedLocation;
   bool debugInfo;
 
+  int controlFlowDepth = 0;
```

Comment:
please add a comment saying what this is used for

Reply:
Done